src/ensembl/src/shared/inline-svg/InlineSvg.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import faker from 'faker';
+
+import apiService from 'src/services/api-service';
+
+import InlineSVG from './InlineSvg';
+
+const sampleSVG = `
+<svg viewBox="0 0 32 32">
+  <rect width="1" height="1" />
+</svg>
+`;
+
+describe('<InlineSVG />', () => {
+  beforeEach(() => {
+    jest
+      .spyOn(apiService, 'fetch')
+      .mockImplementation(() => Promise.resolve(sampleSVG));
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  it('fetches an svg and sets it as inline element', async () => {
+    const src = faker.random.image();
+    const wrapper = mount(<InlineSVG src={src} />);
+
+    // FIXME: currently, the line below produces a warning (because of React state update)
+    // it can be fixed by wrapping it into the asynchronous act function
+    // introduced in react v.16.9.0 alpha, like so:
+    // await act(async () => {
+    //   await new Promise((resolve) => setTimeout(resolve, 0));
+    // })
+    await new Promise((resolve) => setTimeout(resolve, 0));
```

Comment:
Regarding the Fixme above, we can upgrade to React v.16.9.0 alpha, if we are feeling adventurous (I am 😄 ), and the warning that is currently displayed in the console:

![image](https://user-images.githubusercontent.com/6834224/58187708-3a417480-7caf-11e9-961f-e101cd0fdf6b.png)

should go away. What do you say?

Reply:
I think that it would be *too adventurous* to upgrade to an alpha version.

Reply:
According to their changelog, the **only** change that happened in 16.9.0 alpha compared to 16.8.whatever was updating their testing utility.

Reply:
I still think we should not do this just to get rid of a test warning. And we probably need to have a guideline/consensus on what versions (alpha, beta, rc, etc.) of the libraries we can use in the project.

Reply:
As agreed, will not upgrade React to 16.9.0.alpha in this PR. If we get bothered by this warning, we'll update React in a separate PR.